src/lib.rs

```diff
@@ -109,7 +109,14 @@ crate. Pretty much everything else is for ergonomics.
 #![deny(clippy)]
 #![cfg_attr(feature = "ci", deny(warnings))]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(not(feature = "std"), feature(core_intrinsics, extern_prelude))]
```

Comment:
This is the offending line. Unstable features can only be used on nightly.
Activating them whenever `std` is absent prevents any use of `no_std` on stable by triggering a compile error.